CSS/icons-style.css

```diff
@@ -0,0 +1,40 @@
+@font-face {
+  font-family: 'icons';
+  src:  url('../fonts/fonts-icon/icons.eot?4yyeq2#iefix') format('embedded-opentype'),
+    url('../fonts/fonts-icon/icons.ttf?4yyeq2') format('truetype'),
+    url('../fonts/fonts-icon/icons.woff?4yyeq2') format('woff'),
+    url('../fonts/fonts-icon/icons.svg?4yyeq2#icons') format('svg');
+  font-weight: normal;
```

Comment:
You are using relative path to your local resources but you didn't include any fonts in your repository. It could be the reason why icons do not show.